### ISSUE TITLE
Implicit prefetch is not detected

### ISSUE THREAD

**Issue by richardhapb** [richardhapb/django-check#1]
Example

```python
dishes = list(
    michelin.dishmfes.select_related("recipe", "ticker", "chef")
    .prefetch_related("patterns__pattern_exits__windowed_mfes")
)

for dish in dishes:
    for pattern in dish.patterns.all():
        # This line triggers a false positive, `pattern_exits` was prefetched implicitly before.
        exits = list(pattern.pattern_exits.all()) 
```

**Comment by richardhapb** [comment]
Fixed in commit 2895df7
